recipe/meta.yaml

```diff
@@ -31,7 +31,6 @@ requirements:
     - kombu >=5.2.3,<6.0
```

Comment:
Oh yeah, I wondered about this. `grayskull` comes up with `importlib-metadata >=1.4.0  # [py<38]` and no  `noarch: python`, but since we want noarch this makes sense. Thanks!

Reply:
add importlib-metadata dependency